python/src/trezorlib/_rlp.py

```diff
@@ -0,0 +1,55 @@
+# inspired by core/src/trezor/crypto/rlp.py
+
+import typing as t
+from collections.abc import Sequence
+
+if t.TYPE_CHECKING:
+    RLPItem = t.Union[t.Sequence["RLPItem"], bytes, int]
+
+
+def _byte_size(x: int) -> int:
+    if x < 0:
+        raise ValueError("only unsigned ints are supported")
+    return (x.bit_length() + 7) // 8
+
+
+def _int_to_bytes(n: int) -> bytes:
+    """Convert python objects to bytes can be deserialize with converters"""
```

Comment:
nit: kinda hard to parse

Reply:
yeah that would be because it comes from a different function
eba3577832a788ab7cc56008551410f88b71224b